### PR TITLE
Fix python code gen bug in handling "." in property name.

### PR DESCRIPTION
**Pull request by xingwu1**

### CODE REVIEW

**Comment by xingwu1** [comment]
This is the fix for issue #1143

File: AutoRest/TestServer/swagger/paging.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   "paths": {
     "/paging/single": {
       "get": {
-        "x-ms-pageable": { "nextLinkName": "nextLink", "itemName": "values" },
+        "x-ms-pageable": { "nextLinkName": "next.Link", "itemName": "values" },
```
Comment (amarzavery) [inline AutoRest/TestServer/swagger/paging.json]: This will cause a change in all the languages. Have you generated code for all the languages?

Comment (xingwu1) [inline AutoRest/TestServer/swagger/paging.json]: @amarzavery Yes.

Comment (tbombach) [inline AutoRest/TestServer/swagger/paging.json]: @xingwu1 Could the '.' scenario be tested in its own scenario in the test server? It makes it harder to know when this specific behavior breaks if it's part of an existing paging test. I would prefer to see it as a test in a new path in a swagger file, not a modification of an existing one.

**Comment by xingwu1** [comment]
@tbombach I have added a new endpoint with the additional model to repro the bug.

**Comment by tbombach** [comment]
Thanks @xingwu1. Merged

**Pull request merged**